square.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html>
+<head>
+<style>
+    .square {
+    width: 150px;
+    height: 200px;
+    Background-color: red;
+    border: 2px mediumvioletred;
+    }
+</style>
```

Comment:
This a cool idea.  To use this style, you'll need a `div` tag with a `class="square"` attribute, like this: